### ISSUE TITLE
python3: can't open file 'run.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by leliile** [evmer/scribd-downloader#12]
C:\Users\zql-->python3 run.py https://fr.scribd.com/book/495509558/Comprendre-BITCOIN-en-seulement-2H
python3: can't open file 'run.py': [Errno 2] No such file or directory

**Comment by leliile** [comment]
I don't know what went wrong, didn't open browser serious account after running

**Comment by tomeli5n** [comment]
> C:\Users\zql-->python3 run.py https://fr.scribd.com/book/495509558/Comprendre-BITCOIN-en-seulement-2H python3: can't open file 'run.py': [Errno 2] No such file or directory

could you run this command?

`dir`

It seems like you're not running the command in the project folder

**Comment by rankpuertorico** [comment]
Newb - can someone exlplain the solution? I am experiencing the same error. Ive tried running local on mac terminal, and also as a replit.

**Comment by thieuphong220203** [comment]
> Newb - can someone exlplain the solution? I am experiencing the same error. Ive tried running local on mac terminal, and also as a replit.

you can find a folder "scribd-downloader-main" (the folder you download from git) and then you go to that folder you will see the file name "run.py", I do not use MAC but I think it's the same on Window. Now you open terminal on Mac and use the command: cd + path of the folder contain "run.py".
example: my run.py on the folder with this path: "D:\tool\scribd-downloader-main"
<img width="576" alt="Untitled" src="https://github.com/evmer/scribd-downloader/assets/120488704/2df492e6-4d44-41e7-a637-739185923a7d">

I go to terminal and input like this: (This just for window), you can search "how to get path on Mac" on youtube 
<img width="460" alt="image" src="https://github.com/evmer/scribd-downloader/assets/120488704/1c53d38a-0f2f-4b25-8834-5f815ed09e52">

When you go to the folder contain "run.py" you can download the ebook from terminal.
Hope it can help you
